### PR TITLE
Small improvements from Buckinghamshire work

### PR DESCRIPTION
**Pull request by davea**
A collection of little improvements that have shaken out of the Buckinghamshire cobrand.

### CODE REVIEW

File: perllib/FixMyStreet/DB/Result/Problem.pm [hunk 1]
```diff
@@ -660,16 +660,21 @@ sub body {
...
+                '<a href="' . $_->url($c) . '">' . $name . '</a>';
+            } else {
+                $name;
+            }
+        } values %$bodies;
+        if ( scalar @body_names > 2 ) {
+            my $l = scalar @body_names;
+            $body = join( _(', ', ), @body_names[0..$l-2]);
+            $body = join( _(', and '), ($body, $body_names[$l-1]));
+        } else {
```
Comment (struan) [inline perllib/FixMyStreet/DB/Result/Problem.pm]: `splice @body_names, 0, -1` and `$body_names[-1]` might be a more perly way to do this.

File: perllib/FixMyStreet/DB/Result/Problem.pm [hunk 2]
```diff
@@ -660,16 +660,21 @@ sub body {
...
+            if ($c and FixMyStreet->config('AREA_LINKS_FROM_PROBLEMS')) {
+                '<a href="' . $_->url($c) . '">' . $name . '</a>';
+            } else {
+                $name;
+            }
+        } values %$bodies;
+        if ( scalar @body_names > 2 ) {
+            my $l = scalar @body_names;
+            $body = join( _(', ', ), @body_names[0..$l-2]);
+            $body = join( _(', and '), ($body, $body_names[$l-1]));
```
Comment (dracos) [inline perllib/FixMyStreet/DB/Result/Problem.pm]: Can we not have a comma in translation, so it's same as row below.

Comment (davea) [reply perllib/FixMyStreet/DB/Result/Problem.pm]: Why do you hate Oxford commas Matthew 😢

Comment (dracos) [reply perllib/FixMyStreet/DB/Result/Problem.pm]: No keep the comma! Just not in the translation

Comment (dracos) [reply perllib/FixMyStreet/DB/Result/Problem.pm]: Also, don't translated `', '` the row above, just spotted, not sure that's much use ;)

**Pull request merged**